sdk/node/src/api/transactions.js

```diff
@@ -385,6 +397,37 @@ const transactionsAPI = (client) => {
     },
 
     /**
+     * sign - Sign a single transaction.
+     *
+     * @param {Object} template - A single transaction template.
+     * @param {objectCallback} [callback] - Optional callback. Use instead of Promise return value as desired.
+     * @returns {Object} Transaction template with all possible signatures added.
+     */
+    sign: (template, cb) => finalize(template)
+      .then(finalized => client.signer.sign(finalized, cb)),
+
+    /**
+     * signBatch - Sign a batch of transactions.
+     *
+     * @param {Array<Object>} templates Array of transaction templates.
+     * @param {objectCallback} [callback] - Optional callback. Use instead of Promise return value as desired.
+     * @returns {BatchResponse} Tranasaction templates with all possible signatures
+     *                         added, as well as errors.
+     */
+    signBatch: (templates, cb) => finalizeBatch(templates)
+      .then(finalized => client.signer.signBatch(finalized.successes, cb)),
+
+    /**
+     * finalize - Finalize a single transaction
+     */
+    finalize,
```

Comment:
I agree - I saw https://github.com/chain/chain/blob/ca/docs/future/protocol/specifications/ca-api.md#js-2 accessing it directly, but I have trouble thinking of why a user would want to call this. I'll take these out.